packages/relay/src/lib/services/cacheService/cacheService.ts

```diff
@@ -168,11 +171,11 @@ export class CacheService {
    * @param {string} [requestIdPrefix] - The optional request ID prefix.
    * @returns {Promise<any>} A Promise that resolves with the cached value or null if not found.
    */
-  public getSharedWithFallback(key: string, callingMethod: string, requestIdPrefix?: string): any {
```

Comment:
QQ: Do we need to specify such details to the user of this class? I don't think the person using this class will care about implementation details, he wants to just retrieve something from the cache.

I think that's It would be simpler if this was named just `get` or `getAsync`. Especially, since we only provide this one as a public method for getting something from the cache, we don't need to specify details in the method's name.

Reply:
You are probably right that it doesn't need to specify details, so I will rename it to getAsync, since the methods is asynchronous